apps/docs-app/app/components/f/form/search.gts

```diff
@@ -0,0 +1,334 @@
+            />
+          </:example>
+          <:api as |Args|>
+            <Args.String
+              @name="class"
+              @description="The class to apply to the input. Note that this is not an argument but rather a class applied directly to the input"
+              @value={{this.class}}
+              @onInput={{fn this.update "class"}}
+              @options={{this.classOptions}}
+            />
+            <Args.Bool
+              @name="basic"
+              @defaultValue={{false}}
+              @description="When true, the border will be removed"
+              @value={{this.basic}}
+              @onInput={{fn this.update "basic"}}
+            />
+            <Args.String
+              @name="binding"
+              @description="Create a two-way binding with the value"
+              @value={{this.model.property}}
+              @onInput={{fn this.update "model.property"}}
+            />
+            <Args.Bool
+              @name="clearable"
+              @defaultValue={{false}}
+              @description="When true, adds a clear button"
+              @value={{this.clearable}}
+              @onInput={{fn this.update "clearable"}}
+            />
+            <Args.Bool
+              @name="disabled"
+              @defaultValue={{false}}
+              @description="When true, the input will be disabled"
+              @value={{this.disabled}}
+              @onInput={{fn this.update "disabled"}}
+            />
+            <Args.Bool
+              @name="hideSearchIcon"
+              @defaultValue={{false}}
+              @description="When true, the search icon will be hidden"
+              @value={{this.hideSearchIcon}}
+              @onInput={{fn this.update "hideSearchIcon"}}
+            />
+            <Args.Bool
+              @name="loading"
+              @defaultValue={{false}}
+              @description="When true, the icon will be replaced with a loading spinner"
+              @value={{this.loading}}
+              @onInput={{fn this.update "loading"}}
+            />
+            <Args.Number
+              @name="minCharacters"
+              @defaultValue={{1}}
+              @description="The minimum number of characters needed to start a search"
+              @value={{this.minCharacters}}
+              @onInput={{fn this.update "minCharacters"}}
+            />
+            <Args.String
+              @name="noResultsLabel"
+              @defaultValue={{"No results found"}}
+              @description="The label to display when no results are found"
+              @value={{this.noResultsLabel}}
+              @onInput={{fn this.update "noResultsLabel"}}
+            />
+            <Args.String
+              @name="placeholder"
+              @defaultValue={{"Search"}}
+              @description="The placeholder text"
+              @value={{this.placeholder}}
+              @onInput={{fn this.update "placeholder"}}
+            />
+            <Args.Action
```

Comment:
Can you move this action to the bottom of the list, like in the component interface?